README.md

```diff
@@ -1,8 +1,33 @@
-## Generate something from GIR
+# GIR
+
+The `GIR` is used to generate both sys level and Rust-user API level.
+
+## Generate FFI (sys level)
+
+You first need to get the Gir file corresponding to the project you want to bind. You can get them from [here](https://github.com/gtk-rs/gir-files) or directly on [ubuntu website](http://packages.ubuntu.com/) (for example: http://packages.ubuntu.com/zesty/amd64/libgtk-3-dev).
+
+Then you need to write a TOML file you'll pass to gir (you can take a look to [gtk-rs/sys/gir-gtk.toml](https://github.com/gtk-rs/sys/blob/master/conf/gir-gtk.toml) to see an example).
+
+*This part needs more information about this toml file*
+
+When you're ready, let's generate the FFI part:
 
-Running:
 ```shell
-cargo run --release -- -c Gir_Gtk.toml -d ../gir-files -o gtk
+cargo run --release -c YourGirFile.toml -d ../gir-files -m sys -o the-output-directory-sys
```

Comment:
YourGirFile.toml has minor conflict with "Then you need to write a TOML file" before.
Also API level says "You'll now have to write your own GIR file"